node/device/transport/http/lib/http.js

```diff
@@ -295,7 +295,7 @@ Http.prototype.getReceiver = function getReceiver (done) {
     this._receiver = new HttpReceiver(this._config, this._http);
   }
   
-  done(this._receiver);
+  done(null, this._receiver);
```

Comment:
Good catch.  But don't we also need to make similar changes in device/core/lib/client.js!Client#getReceiver, http_simulated.js, unit tests, etc.?